src/components/blocks/latestItemsBlock/LatestItemsBlock.tsx

```diff
@@ -0,0 +1,44 @@
+import { LinkWithQuery } from "@/components/LinkWithQuery";
+import { TLatestItem } from "@/types";
+import { formatDate } from "@/utils/timedate";
+
+export interface IProps {
+  latestItems: TLatestItem[];
+}
+
+export const LatestItemsBlock = ({ latestItems: latestItems }: IProps) => {
+  const ITEM_LIMIT = 3;
+  const limitedLastItems = latestItems.slice(0, ITEM_LIMIT);
+
+  return (
+    <aside aria-labelledby="latest" className="w-[321px] h-[520px] bg-[#E8EBED] p-4 shadow flex flex-col">
```

Comment:
Request: avoid defining colours in this manner, we should have most defined in the `theme.css` file, this colour looks similar enough to the current `surface-heavy` which would mean the class becomes `bg-surface-heavy`